dspace/config/spring/api/external-services.xml

```diff
@@ -20,10 +20,17 @@
         <property name="apiKey" value="${sherpa.romeo.apikey}"/>
 
     </bean>
-    <bean class="org.dspace.external.provider.impl.OrcidV2AuthorDataProvider" init-method="init">
-        <constructor-arg value="${orcid.api.url}"/>
-        <property name="sourceIdentifier" value="orcidV2"/>
+    <bean class="org.dspace.external.provider.impl.OrcidV3AuthorDataProvider" init-method="init">
+        <property name="sourceIdentifier" value="orcid"/>
```

Comment:
It looks like this change from `orcidV2` to `orcid` for the `sourceIdentifier` will require minor updates to Angular UI's `en.json5` language pack... In the UI, I'm seeing missing labels in the External Services features when it comes to ORCID.

So, for example, [this line](https://github.com/DSpace/dspace-angular/blob/main/src/assets/i18n/en.json5#L2971) needs to be updated to be: 
```
"submission.import-external.source.orcid": "ORCID",
```

The same seems to be true for all other i18n keys which include `orcidV2` in them.  The key names all need to be updated to just use `orcid`.

This is a minor change overall, but we should clean this up in a small PR to the Angular UI.

Reply:
Hi @tdonohue I am taking charge of this small PR on the Angular UI. Let you know soon.

Reply:
Hi @tdonohue , angular PR opened
https://github.com/DSpace/dspace-angular/pull/954